Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rust:1.56 AS builder
+
+WORKDIR /app
+
+COPY . ./
+
+RUN cargo build --release
+
+FROM ubuntu AS final
+
+COPY --from=builder /app/target/release/agora /usr/local/bin/agora
+
+ENV PATH="/usr/local/bin:$PATH"
```

Comment:
Is `/usr/local/bin` not in the path by default?

```suggestion
```

Reply:
Fixed.